source/textUtils.py

```diff
@@ -1,9 +1,8 @@
 # -*- coding: UTF-8 -*-

```

Comment:
You might as well remove this if you're changing the header, I don't think gettext fails on this any more.

```suggestion
```